components/resources/commands.grdp

```diff
@@ -398,7 +398,7 @@
     Move group to new window
   </message>
   <message name="IDS_IDC_CLOSE_DUPLICATE_TABS">
-    Close duplicate tabs
+    Close Duplicate Tabs
```

Comment:
I changed it otherwise it would look off in these menus.

Reply:
I think we might want to add a title case version and non-title case version. I'm assuming you're using MacOS, where all menu items are title case. On Windows and Linux menus are sentence case:
![image](https://github.com/brave/brave-core/assets/7678024/00e78974-b028-4d09-a6b5-93fac8cb2491)

Reply:
I think `<if expr="use_titlecase">` is the normal check :smile:

Reply:
Yes I just found it too, will update.

Reply:
Fixed in https://github.com/brave/brave-core/pull/22041/commits/616801d07112a1bc1f74cb5102a61c0e58ab5f48